src/keywords.rs

```diff
@@ -941,6 +941,7 @@ pub const RESERVED_FOR_TABLE_ALIAS: &[Keyword] = &[
     // Reserved for Snowflake table sample
     Keyword::SAMPLE,
     Keyword::TABLESAMPLE,
+    Keyword::FROM,
```

Comment:
is this change related to the PR (wasn't clear to me if so)?

Reply:
Yes.
When removing this as a keyword the following error occurs:
`Error during parsing: ParserError("Expected: ), found: t1 at Line: 9, Column: 13")`